massgov/mayflower
# React/Dp-12303: InputSlider.

## Description
Adds CompoundSlider, InputSlider, and Error components. Also adds the [react-compound-slider package](https://github.com/sghall/react-compound-slider). Adds FormContext support, changes the way Input renders to reduce re-renders on context changes.

## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-12303)

## Steps to Test
1. Pull the branch, run storybook, verify the atom at atoms/forms/InputSlider works.

## Screenshots
![screencapture-localhost-6006-iframe-html-2019-01-29-12_27_50](https://user-images.githubusercontent.com/5257660/51927319-55ffd280-23c1-11e9-85a8-4b740059cab1.png)
